### ISSUE TITLE
ImportError: No module named importlib_metadata

### ISSUE THREAD

**Issue by nodecentral** [mcastorina/graph-cli#47]
Hi

I’ve just installed the latest version of graph_cli (1.11) and when I run it - I always get the following error message..

````
pi@raspberrypi:~ $ graph
Traceback (most recent call last):
  File "/home/pi/.local/bin/graph", line 10, in <module>
    sys.exit(main.main())
  File "/home/pi/.local/lib/python2.7/site-packages/graph_cli/main.py", line 8, in main
    args = options.parse_args()
  File "/home/pi/.local/lib/python2.7/site-packages/graph_cli/options.py", line 390, in parse_args
    import importlib_metadata as metadata
ImportError: No module named importlib_metadata
````
Any ideas ?

**Comment by mcastorina** [comment]
Shoot. Yeah, I recently added that for getting the version string. Try installing `importlib-metadata` to see if that fixes it.

```
pip install importlib-metadata
```

I might have to add it as a dependency in pip, although I don't really want to do that for such a simple feature.

**Comment by nodecentral** [comment]
Thanks, that helped to address that specific error message, but we now have a new one :-( - https://github.com/mcastorina/graph-cli/issues/48

**Comment by mcastorina** [comment]
Alright, I removed importlib-metadata in v0.1.12. Thanks for finding and reporting this and sorry for all the back and forth.

Looking at the new issue now..

**Comment by mcastorina** [comment]
Closing this since it is resolved (and starting with v0.1.12 has no importlib-metadata dependency).
